src/UI/Settings/NetImport/Options/NetImportOptionsView.js

```diff
@@ -20,7 +16,11 @@ var view = Marionette.ItemView.extend({
 	},
 	
 	onShow : function() {
+		var params = new URLSearchParams(window.location.search.slice(1));
```

Comment:
![image](https://cloud.githubusercontent.com/assets/23205277/22866946/33a84590-f14c-11e6-8f60-fe9e72b6f70a.png)

I'm not sure if I need to guard var URLSearchParams = require('url-search-params');

with the a check like theone in the image.. and im not sure what this was referring to or if I need anything related to it:
![image](https://cloud.githubusercontent.com/assets/23205277/22866960/5feaa116-f14c-11e6-9660-03efa396fb58.png)


I tested in Chrome/IE... and it seems to be working.